app/repositories/meteors.repository.test.ts

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
```

Comment:
I don't think there is any point in testing repository layer. Typically these are queries to a database and this layer does not and should not contain any business logic

Reply:
Agree, but I was just trying to train testing to be more familiar with it. That's why I decide to cover this layer with tests.